packages/e2e-tests/specs/editor/blocks/gallery.test.js

```diff
@@ -52,6 +61,50 @@ describe( 'Gallery', () => {
 		expect( await getEditedPostContent() ).toMatch( regex );
 	} );
 
+	it( 'gallery caption can be edited', async () => {
+		const galleryCaption = 'Tested gallery caption';
+
+		await insertBlock( 'Gallery' );
+		const filename = await upload( '.wp-block-gallery input[type="file"]' );
+
+		await getUploadedFileFigure( filename );
+
+		await page.click( '.wp-block-gallery>.blocks-gallery-caption' );
+		await page.keyboard.type( galleryCaption );
+
+		expect( await getEditedPostContent() ).toMatch(
+			new RegExp( `<figcaption.*?>${ galleryCaption }</figcaption>` )
+		);
+	} );
+
+	it( "uploaded images' captions can be edited", async () => {
+		await insertBlock( 'Gallery' );
+		const filename = await upload( '.wp-block-gallery input[type="file"]' );
+
+		const figureElement = await getUploadedFileFigure( filename );
+
+		await figureElement.click();
+
+		expect(
+			await figureElement.evaluate( ( element ) =>
+				element.classList.contains( 'is-selected' )
+			)
+		).toBeTruthy();
```

Comment:
Do we need this check?

Reply:
I just wanted to check if the element was being correctly selected, but I guess this can be checked in the last assertion anyway.